### DISCUSSION TITLE
reinstall bootloader on emmc without loss data after failed kernel update

### DISCUSSION THREAD

**Discussion by diktus** [ophub/amlogic-s9xxx-armbian#2553] [Amlogic]
I have h96-max-x3 running on emmc. after update using apt update, seems kernel is updating but after reboot. system can not boot with the error as attach. how to reinstall the bootloader without format the other partition? i have many apps and database running in this machine.

Thank You
<img width="1435" alt="Screenshot 2024-09-16 at 21 52 12" src="https://github.com/user-attachments/assets/1004d9ac-78b6-4659-be36-2ea1f56b8a83">

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/tree/main/documents#10-updating-armbian-kernel


`armbian-update -s`
